hyperspy/misc/date_time_tools.py

```diff
@@ -19,23 +19,15 @@
 import numpy as np
 import datetime
 from dateutil import tz, parser
+import pytz
```

Comment:
I used pytz for convenience, dateutil.timezone looked to be quite limited. Yeah, adding one more library is not great and I guess it should be possible to do the same with dateutil.
I will not have time to do it in the next few days, so please go ahead if like. Otherwise I can also do it later.